### PR TITLE
Redux logging

### PR DESCRIPTION
**Pull request by mixcas**
I added LogRocket to log all moves and it's fucking amazing. Login info is in this project trello board.

### CODE REVIEW

**Comment by mikeyrayvon** [comment]
what do i do to test this?

**Comment by mixcas** [comment]
Idk :/
its working tho

File: src/firebase.js [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
-import { reactReduxFirebase } from 'react-redux-firebase'
+import { reactReduxFirebase } from 'react-redux-firebase';
+import LogRocket from 'logrocket';
+
+// Set middlewares
+let middlewares = [];
+
+if(process.env.NODE_ENV === 'production') {
+  // Logrocket
+  LogRocket.init('2xmblo/andamiaje');
```
Comment (varffs) [inline src/firebase.js:11]: Is this not a secret?

**Comment by mixcas** [comment]
Not anymore! Jk no, it's ok :)

**Pull request merged**